ff/src/fields/mod.rs

```diff
@@ -418,6 +443,11 @@ pub trait PrimeField:
         Self::Params::T_MINUS_ONE_DIV_TWO
     }
 
+    /// Returns the trace minus one divided by two.
+    fn modulus() -> Self::BigInt {
+        Self::Params::MODULUS
+    }
```

Comment:
This already exists as the `characteristic` method on `Field`.
```suggestion
```

Reply:
I think in my use case I needed it to be in the bigint format. I think `&[u64]` is problematic because then it requires a copy... which is not terribly expensive but...

Reply:
I see, since it's going to be a constant maybe it's not copied at all? Is this mainly meant to be syntactic sugar for `<F as PrimeField>::Params::MODULUS`?

Reply:
Hmm, I'm not entirely clear on whether or not its copied. 
> Is this mainly meant to be syntactic sugar for <F as PrimeField>::Params::MODULUS?
Yes, I don't think this particular method should be an issue given we also define 
```
/// Returns the modulus minus one divided by two.
    fn modulus_minus_one_div_two() -> Self::BigInt {
        Self::Params::MODULUS_MINUS_ONE_DIV_TWO
    }
```
etc

Reply:
Ok yeah, that makes sense.